pages/_error.js

```diff
@@ -0,0 +1,126 @@
+import styled from 'styled-components'
+import fetch from 'node-fetch'
+import Link from 'next/link'
+import PostItemList from '../components/PostItemList'
+import Layout from '../components/Layout'
+import SearchBar from '../components/SearchBar'
+
+const Container = styled.div`
+  max-width:1032px;
+  margin:auto;
+  padding:${(props) => props.theme.space * 2}px;
+
+`
+const ErrorWrapper = styled.section``
+const ErrorCode = styled.h1`
+  font-size:152px;
+  margin:0;
+  text-align:left;
+  font-family: ${(props) => props.theme.fontFamilyText};
+`
+const ErrorDescription = styled.h2`
+  font-size:38px;
+  margin:0;
+  text-align:left;
+  font-family: ${(props) => props.theme.fontFamilyText};
+`
+const PostsWrapper = styled.section`
+  background:#fafafa;
+`
+
+const Search = styled.div`
+  padding-top:60px;
+`
+
+const Paragraph = styled.p`
+  color:${(props) => props.theme.color.ultraGray};
+`
+
+const Error = ({ categories, posts, statusCode }) => {
+  const heroNews = posts ? [posts[1], posts[2], posts[3]] : []
+  const errors = [
+    {
+      code: 404,
+      description: 'We couldn’t find this page.',
+      message: 'Maybe it’s out there, somewhere...',
+      displayPosts: true,
+      displayHome: true,
+    },
+    {
+      code: 503,
+      description: 'Internal Server Error.',
```

Comment:
I would change the message description for something like: **"We are having troubles now"** or something similar